app/assets/stylesheets/lib/_bootstrap_overrides.scss

```diff
@@ -0,0 +1,7 @@
+.breadcrumb > .active {
```

Comment:
Previously we only used `lib` for SASS functions/variables/mixins that didn't actually output any CSS. Perhaps this should be a `helper` or something? 

This would also clear up the load order weirdness.